.ci/docs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+NAME=$0
+COMMAND=$1
+
+function usage {
+    echo "usage: $NAME <command>"
+    echo
+    echo "  install  Install dependencies for building docs"
+    echo "  check    Check documentation for broken links"
+    echo "  run      Build documentation"
+    echo "  upload   Upload documentation to gh-pages branch"
+    exit 1
+}
+
+if [[ "$COMMAND" == "install" ]]; then
+    conda install --quiet jupyter matplotlib numpy pillow scipy
+    pip install "sphinx<1.7" nbsphinx numpydoc guzzle_sphinx_theme
+    pip install -e .
+elif [[ "$COMMAND" == "check" ]]; then
+    sphinx-build -b linkcheck -v -D nbsphinx_execute=never docs docs/_build
+elif [[ "$COMMAND" == "run" ]]; then
+    git clone -b gh-pages https://github.com/nengo/nengo-loihi.git ../docs
+    sphinx-build -vW docs ../docs
```

Comment:
Oh I got this from your nengo-dl stuff; we clone another copy of the repo to `../docs` and start on the gh-pages branch so that when we build we are essentially updating the HTML files there rather than wiping the history every time. I didn't take all of the versioning stuff from Nengo DL yet, but probably will eventually, just did this halfway measure with the time I had. I do think other projects should switch to this, assuming that the image files don't change too frequently as that would bloat up the size of the repo.

Reply:
It also means we don't need the `ghp-import` dependency, which is kinda good I guess